debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -166,17 +166,17 @@ public static EventProcessingFailureHandlingMode parse(String value) {
         /**
          * Represent binary values as byte array
          */
-        BYTES("bytes", () -> SchemaBuilder.bytes()),
+        BYTES("bytes", SchemaBuilder::bytes),
```

Comment:
Same comment as above: all those should rather go into a separate housekeeping commit (mentioning it for next time, no need to change now).

Reply:
okay. will do next time